### DISCUSSION TITLE
Handling of outdated noise data

### DISCUSSION THREAD

**Discussion by JohannaOtt** [INSPIRE-MIF/helpdesk#219] [General]
Hi,

We are currently publishing the noise data available in [this WFS](https://geodienste.komm.one/ows/services/org.546.5e955aac-82bd-4eac-8fa8-4a50659fe491_wfs?SERVICE=WFS&REQUEST=GetCapabilities&VERSION=2.0.0).
It contains noise contours valid from 2017 to 2021.
Now, new noise contours for the years after are available.
What is the recommended way of handling the old data?
- Publish a new service for the new data and keep the old service as it is?
- Publish a new service fot the new data and delete the old services
- Replace the data in the service so that users can still use it via the same URL?
- Any other best practices?

Thank you and all the best

Johanna

**Comment by sMorrone** [comment]
Dear @JohannaOtt,
if still managed by data providers, historical data must also be made available under INSPIRE, so if this is your case you should create an additional service. There are no specific best practices that I can mention here. You could for instance make the old data available  only as predefined datasets - e.g., via Atom feed - and use the WFS service for the new data or conversely decide to make both datasets available via WFS services, depending on the specific needs. In any case, ,it is essential that everything is properly documented in the metadata.
